share/spice/people_in_space/people_in_space.js

```diff
@@ -1,102 +1,62 @@
-
-            //format launchdate per locale
-            //people[i]["launchdate"] = launchdate.toLocaleDateString("", {year: "numeric", month: "short", day: "numeric"});
-            people[i].launchdate = months[launchdate.getMonth()] + " " + launchdate.getDate() + ", " + launchdate.getFullYear();
-
-            //rename title because it conflicts with template
-            people[i].position = people[i].title;
-        }
-
-        //The icon template group won't show anything if there are zero items, but we want to show "no one is in space" in that case
-        //So we'll build an object conditionally with either "icon" or "text" template groups, and then call Spice.add()
-        var o = {
+        Spice.add({
             id: "people_in_space",
             name: "Answer",
+            data: api_result.people,
             meta: {
-                itemType: (api_result.number === 1 ? "Person" : "People"),
+                itemType: (api_result.number === 1 ? "Person in Space" : "People in Space"),
                 sourceName: "People in Space",
                 sourceUrl: "http://www.howmanypeopleareinspacerightnow.com/"
-            }
-        };
-        if (people.length > 0) {
-            o.data = people;
-            o.normalize = function(item) {
-                return {
-                    url: item.biolink,
-                    title: item.name,
-                    icon: DDG.settings.region.getSmallIconURL(item.country_code)
+            },
+            normalize: function(item) {
+           		return {
+           			daysElapsed: moment().diff(item.launchdate, 'days'),
+           			icon: DDG.settings.region.getLargeIconURL(codes[item.country.toLowerCase()]),
+                    twitter: item.twitter.replace(/https?:\/\/twitter.com\//,'')
                 };
-            };
-            o.sort_fields = {
-                launchdate: function(a, b) {
-                    //The dates are YYYY-MM-DD strings, so we can just use alpha sort.
-                    return a.launchdate < b.launchdate ? -1 : (a.launchdate > b.launchdate ? 1 : 0);
+            },
+           templates: {
+                item: 'base_item',
+    			item_detail: Spice.people_in_space.detail,
+                options: {
+                    content: Spice.people_in_space.content,
                 }
             }
-            o.sort_default = "launchdate";
-            o.templates = {
-                group: "icon",
-                detail: false,
-                item_detail: false,
-                options:{
-                    footer: Spice.people_in_space.footer,
-                    moreAt: true
-                },
-                variants: {
-                    tileFooter: '3line'
-                }
-            };
-        } else {
-            o.data = api_result;
-            o.templates = {
-                group: "base",
+        });
+   });
+} else {
```

Comment:
Not a huge deal, but to avoid a bit of duplication, you could just use the `if/else` to modify some variables: `var normalize, meta, templates` and then pass it all into a single `Spice.render()` call. In either case the `id, name, data` are the same.